webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -573,11 +573,15 @@ var userOverride = {
         var KM_ASK_USER_DETAILS = mckMessageService.checkArray(
             appOptions.askUserDetails
         );
-        var KM_PRELEAD_COLLECTION = mckMessageService.checkArray(
-            appOptions.preLeadCollection
-        )
-            ? appOptions.preLeadCollection
-            : [];
+        var KM_PRELEAD_COLLECTION =
+            appOptions.appSettings.collectLead && appOptions.preLeadCollection
+                ? mckMessageService.checkArray(appOptions.preLeadCollection)
```

Comment:
in DB, there are 32,200 customers for whom, collectLead is false, but many of them must already be using the pre-chat form via widget script directly. so it should not break for them but seems like this check will break it for them. @Ojashwa can you verify once?
![Screenshot 2021-03-05 at 6 59 11 PM](https://user-images.githubusercontent.com/23008972/110122006-164c1600-7de5-11eb-944f-aa301b9281a5.png)